### PR TITLE
Update Renderer.php

### PR DESCRIPTION
**Pull request by ThaDafinser**

### CODE REVIEW

File: src/ZfcDatagrid/Renderer/PHPExcel/Renderer.php [hunk 1]
```diff
@@ -103,14 +103,26 @@ public function execute()
                         break;
 
                     case 'ZfcDatagrid\Column\Type\DateTime':
-                        if ($value instanceof \DateTime) {
-                            $value->setTimezone(new \DateTimeZone($col->getType()
-                                ->getOutputTimezone()));
+                        /* @var $dateType \ZfcDatagrid\Column\Type\DateTime */
+                        $dateType = $col->getType();
+                        
+                        if (! $value instanceof \DateTime && ! is_object($value)) {
```
Comment (kokspflanze) [inline src/ZfcDatagrid/Renderer/PHPExcel/Renderer.php]: maybe change `!is_object` to [is_scalar](http://php.net/manual/de/function.is-scalar.php), so check also that its no array and no resource

**Comment by jonathangreco** [comment]
Is this issue can be released ? I need it for december :s

**Comment by ThaDafinser** [comment]
@jonathangreco https://github.com/ThaDafinser/ZfcDatagrid/releases/tag/v1.3.1

File: src/ZfcDatagrid/Renderer/PHPExcel/Renderer.php [hunk 2]
```diff
@@ -103,14 +103,26 @@ public function execute()
...
-                        ->getNumberFormat()
-                        ->setFormatCode(\PHPExcel_Style_NumberFormat::FORMAT_DATE_DATETIME);
+                        
+                        if ($dateType->getOutputPattern()) {
+                            $outputPattern = $dateType->getOutputPattern();
+                        } else {
+                            $outputPattern = \PHPExcel_Style_NumberFormat::FORMAT_DATE_DATETIME;
+                        }
+                        
+                        $cell->$cell->getStyle()
```
Comment (jonathangreco) [inline src/ZfcDatagrid/Renderer/PHPExcel/Renderer.php]: Error, duplicate of $cell call, need to remove one.
@ThaDafinser

Comment (ThaDafinser) [inline src/ZfcDatagrid/Renderer/PHPExcel/Renderer.php]: Done - https://github.com/ThaDafinser/ZfcDatagrid/commit/9134f451ca513171397277852cd39e9e66f5447c

**Pull request merged**